addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -299,15 +305,23 @@ export class Rtc extends Record {
 
         browser.addEventListener("pagehide", () => {
             if (this.state.channel) {
+                const timestamp = new Date().getTime();
                 const data = JSON.stringify({
-                    params: { channel_id: this.state.channel.id },
+                    params: {
+                        channel_id: this.state.channel.id,
+                        recovery_ts: timestamp,
+                    },
                 });
                 const blob = new Blob([data], { type: "application/json" });
                 // using sendBeacon allows sending a post request even when the
                 // browser prevents async requests from firing when the browser
                 // is closed. Alternatives like synchronous XHR are not reliable.
                 browser.navigator.sendBeacon("/mail/rtc/channel/leave_call", blob);
                 this.sfuClient?.disconnect();
+                this.multiTab.broadcast("discuss.rtc/recover", {
+                    channel_id: this.state.channel.id,
+                    recovery_ts: -timestamp,
```

Comment:
`-` looks unintentional (and probably explained why the recovery process doesn't work for me)